app/config/settings.py

```diff
@@ -1133,6 +1133,7 @@ def sentry_before_send(event, hint):
 
 CHALLENGES_DEFAULT_ACTIVE_MONTHS = 12
 CHALLENGE_ONBOARDING_TASKS_OVERDUE_SOON_CUTOFF = timedelta(days=7)
+CHALLENGE_INVOICE_OUTSTANDING_CUTOFF = timedelta(weeks=4)
```

Comment:
Probably good to add the timeunit to the name of this variable for clarity

Reply:
I don't think this is necessary as the value is a `timedelta`, follows the pattern of `CHALLENGE_ONBOARDING_TASKS_OVERDUE_SOON_CUTOFF` above.